lib/network/options.js

```diff
@@ -403,6 +403,7 @@ let allOptions = {
       interpolation: { boolean: bool },
       useImageSize: { boolean: bool },
       useBorderWithImage: { boolean: bool },
+      useImageCoords: { boolean: bool },
```

Comment:
This is not very descriptive name. I suggest (this is up for discussion of course):
```typescript
interface shapeProperties {
  coordinateOrigin?:
    | "center" // default
    | "top-left";
}
```
This way people would have better chance at intuitively guess what it does.

Reply:
This makes sense.